pkg/render/compliance.go

```diff
@@ -601,8 +613,28 @@ func (c *complianceComponent) complianceServerDeployment() *appsv1.Deployment {
 							},
 						},
 					},
+				}},
+				{
+					Name: "manager-cert",
+					VolumeSource: corev1.VolumeSource{
+						Secret: &corev1.SecretVolumeSource{
+							DefaultMode: &defaultMode,
+							SecretName:  ManagerTLSSecretName,
+							Optional:   &optional,
```

Comment:
I don't think we should make this optional. We know if the cert is being rendered or not so we can be confident if this mount is needed or not.